packages/custom-services/package.json

```diff
@@ -17,14 +17,50 @@
     "directory": "packages/custom-services"
   },
   "scripts": {
-    "compile": "tsc --build",
-    "bundle": "tsx esbuild.config.ts",
-    "test": "jest --passWithNoTests",
-    "test:coverage": "jest --coverage --coverageDirectory=./coverage --passWithNoTests",
-    "clean": "rimraf out dist .turbo coverage tsconfig.tsbuildinfo",
+    "compile": "wireit",
+    "bundle": "wireit",
+    "test": "wireit",
+    "test:coverage": "wireit",
+    "clean": "rimraf out dist .wireit coverage tsconfig.tsbuildinfo",
     "clean:all": "npm run clean && rimraf node_modules",
-    "lint": "eslint src",
-    "lint:fix": "eslint src --fix"
+    "lint": "wireit",
+    "lint:fix": "wireit"
+  },
+  "wireit": {
+    "compile": {
+      "command": "tsc --build",
+      "dependencies": ["../apex-parser-ast:precompile"],
+      "output": ["out/**"],
+      "files": ["src/**/*.ts", "tsconfig*.json"]
+    },
+    "bundle": {
+      "command": "tsx esbuild.config.ts",
+      "dependencies": ["compile"],
+      "output": ["dist/**"],
+      "files": ["src/**/*.ts", "esbuild.config.ts"]
+    },
+    "test": {
+      "command": "jest --passWithNoTests --json --outputFile=.wireit/test-results.json || true",
+      "dependencies": ["compile"],
+      "output": [".wireit/test-results.json"],
+      "files": ["src/**/*.ts", "test/**/*.ts", "jest.config.cjs"]
+    },
+    "test:coverage": {
+      "command": "jest --coverage --coverageDirectory=./coverage --passWithNoTests",
+      "dependencies": ["compile"],
+      "output": ["coverage/**"],
+      "files": ["src/**/*.ts", "test/**/*.ts", "jest.config.cjs"]
+    },
+    "lint": {
+      "command": "eslint --color --cache --cache-location .eslintcache src",
+      "dependencies": ["compile"],
+      "files": ["src/**/*.ts", "eslint.config.mjs"],
+      "output": []
+    },
+    "lint:fix": {
```

Comment:
fyi, I did away with these for the monorepo because you can just

`npm run lint -- --fix` (the difference that the command here would only fix src, but the other one would also fix test stuff)